{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"provenance":[],"authorship_tag":"ABX9TyOkWbAXhk1dMkrfuFOXzJRY"},"kernelspec":{"name":"python3","display_name":"Python 3"},"language_info":{"name":"python"}},"cells":[{"cell_type":"code","source":["!pip install pandas"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"jKBjtDXp3Yq8","executionInfo":{"status":"ok","timestamp":1718118889501,"user_tz":-60,"elapsed":8418,"user":{"displayName":"Ezigbo Augustus","userId":"03379521794697506209"}},"outputId":"35201164-d960-4f9c-fce1-a8d3979d947a"},"execution_count":6,"outputs":[{"output_type":"stream","name":"stdout","text":["Requirement already satisfied: pandas in /usr/local/lib/python3.10/dist-packages (2.0.3)\n","Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.10/dist-packages (from pandas) (2.8.2)\n","Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.10/dist-packages (from pandas) (2023.4)\n","Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.10/dist-packages (from pandas) (2024.1)\n","Requirement already satisfied: numpy>=1.21.0 in /usr/local/lib/python3.10/dist-packages (from pandas) (1.25.2)\n","Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.10/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)\n"]}]},{"cell_type":"code","source":["import pandas as pd"],"metadata":{"id":"9b87b_WF27vR","executionInfo":{"status":"ok","timestamp":1718118862460,"user_tz":-60,"elapsed":537,"user":{"displayName":"Ezigbo Augustus","userId":"03379521794697506209"}}},"execution_count":4,"outputs":[]},{"cell_type":"code","execution_count":8,"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":141},"id":"W2NOPBg92WOG","executionInfo":{"status":"error","timestamp":1718118920021,"user_tz":-60,"elapsed":19,"user":{"displayName":"Ezigbo Augustus","userId":"03379521794697506209"}},"outputId":"8dbfc36a-9d06-4088-82b5-faa743ba203e"},"outputs":[{"output_type":"error","ename":"AttributeError","evalue":"module 'pandas' has no attribute 'read_ipynb'","traceback":["\u001b[0;31m---------------------------------------------------------------------------\u001b[0m","\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)","\u001b[0;32m<ipython-input-8-193302cce296>\u001b[0m in \u001b[0;36m<cell line: 1>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mdf\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mpd\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mread_ipynb\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'/content/ydata_profiling.ipynb'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m","\u001b[0;31mAttributeError\u001b[0m: module 'pandas' has no attribute 'read_ipynb'"]}],"source":["df = pd.read_ipynb('/content/ydata_profiling.ipynb')"]},{"cell_type":"code","source":[],"metadata":{"id":"ERYGumOQ3Kq_"},"execution_count":null,"outputs":[]}]}